airbyte-integrations/connectors/source-posthog/source_posthog/schemas/persons.json

```
@@ -13,9 +13,15 @@
         "type": "string"
       }
     },
+    "properties": {
+      "type": "object"
```

Whatever you specify an object in the schema the best practice is to define with additional properties and empty properties.
```suggestion
      "type": ["null", "object"],
      "additionalProperties": true,
      "properties": {}
```